pynamodb/tests/integration/test_transaction_integration.py

```diff
@@ -0,0 +1,263 @@
+from datetime import datetime
+
+import config as cfg
+import pytest
+
+from pynamodb.exceptions import PutError, DoesNotExist
+
+from pynamodb.attributes import NumberAttribute, UnicodeAttribute, UTCDateTimeAttribute, BooleanAttribute
+from pynamodb.connection.transactions import TransactGet, TransactWrite
+
+from pynamodb.models import Model
+
+IDEMPOTENT_PARAMETER_MISMATCH = 'IdempotentParameterMismatchException'
+PROVISIONED_THROUGHPUT_EXCEEDED = 'ProvisionedThroughputExceededException'
+RESOURCE_NOT_FOUND = 'ResourceNotFoundException'
+TRANSACTION_CANCELLED = 'TransactionCanceledException'
+TRANSACTION_IN_PROGRESS = 'TransactionInProgressException'
+
+
+class User(Model):
+    class Meta:
+        region = 'us-east-1'
+        table_name = 'user'
+
+    user_id = NumberAttribute(hash_key=True)
+
+
+class BankStatement(Model):
+
+    class Meta:
+        region = 'us-east-1'
+        table_name = 'statement'
+
+    user_id = NumberAttribute(hash_key=True)
+    balance = NumberAttribute(default=0)
+    active = BooleanAttribute(default=True)
+
+
+class LineItem(Model):
+
+    class Meta:
+        region = 'us-east-1'
+        table_name = 'line-item'
```

Comment:
I think you need the host here

Reply:
the host is set here: https://github.com/pynamodb/PynamoDB/pull/615/files#diff-6bc097399e79abc4d8f5f7d0bc88da30R85